### ISSUE TITLE
Accessibility and vocalisation of cap-js

### ISSUE THREAD

**Issue by lucmuller** [tiagorangel1/cap#47]
IMHO Accessibility could be improved on this well working web component.

1) The HTML is not a _checkbox_ but a _button_ thus it should read in the first place something like "Please click here to prove you're not a human" and not "I'm a human" as there's no real checkbox to check the fact that "I'm a human".
Succeeding messages should read somethink like "You're a human" "You're not a human, please refresh the page and try again".

2) The button it self is subject to state modification, thus it should have a (imho) aria-live="polite" attribute in order to have it's modification vocalised by screen reader.

Btw I'm not a full accessibility expert, so you'd better ask a real expert to confirm the points I'm suggesting, but as far as I know there seems to be accessibility issues as of today.

**Comment by lucmuller** [comment]
Complement : There should be a loaded javascript event trigerred. So that we can alter HTML and/or behavior after cap-js is loaded.

As I read in the current documentation events are trigerred only after events regarding usages of the captcha (solve, error, etc.)

**Comment by tiagozip** [comment]
thanks for your suggestions! i'm also not an expert in accessibility either but i've added better aria-labels and `aria-live="polite"` in widget@0.1.19

**Comment by tiagozip** [comment]
you can now also change the text of aria-labels using the i18n keys
